### ISSUE TITLE
When using KeePassXC integration, finding forms seems to fail with forms in iframes because they are not part of the same document

### ISSUE THREAD

**Issue by jfanjoy** [qutebrowser/qutebrowser#8418]
https://github.com/qutebrowser/qutebrowser/blob/c462cf62e4a2b4bbb89c5072a824ef716c7f7a59/misc/userscripts/qute-keepassxc#L369

This is true even when the iframe form is in focus when the integration is triggered.

**Comment by jfanjoy** [comment]
#8419 addresses this issue, but a build step failed that I don't believe has anything to do with the changes I made. I think that prevents github from linking the PR here.

**Comment by The-Compiler** [comment]
I agree it's unrelated, I triggered a rerun. The linking failed because the `closes #8418` needs to be in the PR description, it won't work in the title.

**Comment by jfanjoy** [comment]
oh ok. I think codeberg wants it in the title and I misread when creating the PR. much obliged.

**Comment by jfanjoy** [comment]
I just pushed another change because I think a comment on the end of one of the lines was causing problems and I forgot to get the change added to the repo. If the build succeeds this time I guess maybe that comment may have been causing problems for ci too.
